### PR TITLE
feat(code reviews): add md sheet on code reviews with info pulled fro…

### PR DESCRIPTION
**Pull request by ryekerjh**
…m Corey Shuman and Ryeker Herndon's code review slide decks.

This introduces the code review process found in the slide decks for the code review S&P meetings. Sorry it's so late!

### CODE REVIEW

File: conventions/code-reviews.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+### Author
+* At Shift3 we have a simple code review process
+
+1. Before you create your Pull Request or tag anyone in anything, double check that you're ready
+
+    Pre-PR checklist:   
+    - [ ] Test the specific code you're submitting
+    - [ ] Use a linter (esLint is probably going to be the standard) to take care of whitespace and styling
+    - [ ] Remove your console. s
```
Comment (carlosvargas) [inline conventions/code-reviews.md]: I think a word got truncated here (I'm assuming it's meant to say `console.logs`?)

File: conventions/code-reviews.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+        * Don’t block the PR unless there is a legitimate bug
+
+        * Frame your feedback as requests, not commands and be kind but direct (avoid using personal pronouns i.e. “Your code sucks…” “My code worked before you messed it up…”)
+
+        * Provide code examples, references, links, etc when applicable
+
+        * Don’t forget to compliment the sumitter on good stuff, too!
+
+2. Mark your review as one of the following: 
+    `Request Changes`: If there are bugs in the code or something unacceptable being introduced to the code base (swearing in comments, console. s etc.), you should block the PR and leave detailed comments as to why. If you block a PR and the author resubmits changes, get to them as quickly as you can and approve the PR once it's addressed
```
Comment (carlosvargas) [inline conventions/code-reviews.md]: > (swearing in comments, console. s etc.)

Another console.logs word that got truncated.

Comment (mwallert) [reply conventions/code-reviews.md]: And sumitter should be submitter, I'm rubbing off my terrible spelling on @ryekerjh :joy:

File: conventions/code-reviews.md (hunk 1 above)
Comment (ryekerjh) [reply conventions/code-reviews.md]: It was meant to refer to all flavor of console dot anything (warn, error, etc) but I didn't know how to convey that. Any suggestions @carlosvargas ?

File: conventions/code-reviews.md (hunk 2 above)
Comment (ryekerjh) [reply conventions/code-reviews.md]: @mwallert DOH! I pushed up that change.

**Comment by ryekerjh** [comment]
closes #47

**Comment by ryekerjh** [comment]
@mwallert @carlosvargas I pushed up a different way of referencing the console and added debuggers. Can you take a look when you have a minute?

File: conventions/code-reviews.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+### Author
+* At Shift3 we have a simple code review process
+
+1. Before you create your Pull Request or tag anyone in anything, double check that you're ready
+
+    Pre-PR checklist:   
+    - [ ] Test the specific code you're submitting
+    - [ ] Use a linter (esLint is probably going to be the standard) to take care of whitespace and styling
+    - [ ] Remove your console references and debuggers
+    - [ ] Make sure your code changes are uner 500 lines of code (break it up into more than 1 PR if it exceeds this)
```
Comment (mwallert) [inline conventions/code-reviews.md]: console references and debuggers love it

Comment (carlosvargas) [reply conventions/code-reviews.md]: oh that's much better than what I was thinking!

**Pull request merged**